scripts/testenv.sh

```
@@ -1,7 +1,11 @@
 #!/usr/bin/env bash
 
-docker compose -f environment/docker-compose.yml --env-file .env up -d db
+if [ -f .test.env ]; then
+    export $(grep -v '^#' .test.env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)